dojo/unittests/tools/test_intsights_parser.py

```diff
@@ -0,0 +1,53 @@
+from django.test import TestCase
+from dojo.tools.intsights.parser import IntSightsParser
+from dojo.models import Engagement, Product, Test
+
+
+class TestIntSightsParser(TestCase):
+    def get_test(self):
+        test = Test()
+        test.engagement = Engagement()
+        test.engagement.product = Product()
+        return test
```

Comment:
I think you can remove this piece and use directly this form in unit tests:

```
findings = parser.get_findings(file, Test())
```

Reply:
Also boilerplate from cookie cutter template...